src/api/python/z3/z3.py

```diff
@@ -9453,15 +9453,15 @@ def _mk_fp_bin_pred(f, a, b, ctx):
     ctx = _get_ctx(ctx)
     [a, b] = _coerce_fp_expr_list([a, b], ctx)
     if z3_debug():
-        _z3_assert(is_fp(a) or is_fp(b), "Second or third argument must be a Z3 floating-point expression")
+        _z3_assert(is_fp(a) or is_fp(b), "First or second argument must be a Z3 floating-point expression")
```

Comment:
`_mk_fp_bin_pred` isn't supposed to be called by a user, it is called by functions such as `fpLT`. When validating these arguments, it is validating the argument _as passed to the caller_, which means we want to ignore `f`.